Remove problematic (read: non-working) checks for several
types which are definitely defined in sys/types.h

--- configure.ac	2016-10-23 13:00:07.000000000 +0200
+++ configure.ac	2016-10-28 20:50:56.881735445 +0200
@@ -274,12 +274,6 @@
 AC_C_CONST
 AC_C_INLINE
 AC_TYPE_SIZE_T
-AC_CHECK_TYPE(uchar, unsigned char)
-AC_CHECK_TYPE(ushort, unsigned short)
-AC_CHECK_TYPE(uint, unsigned int)
-AC_CHECK_TYPE(ulong, unsigned long)
-AC_CHECK_TYPE(longlong, long long)
-AC_CHECK_TYPE(ulonglong, unsigned long long)
 AC_CHECK_TYPE(ssize_t, int)
 AC_CHECK_FUNCS(gettimeofday)
 AC_CHECK_FUNCS(getrusage)
--- src/libjasper/include/jasper/jas_config.h.in	2016-10-23 13:00:59.000000000 +0200
+++ src/libjasper/include/jasper/jas_config.h.in	2016-10-28 20:56:18.232991601 +0200
@@ -139,31 +139,6 @@
 #undef inline
 #endif
 
-/* Define to `long long' if <sys/types.h> does not define. */
-#undef longlong
-
-/* Define to `unsigned int' if <sys/types.h> does not define. */
-#undef size_t
-
-/* Define to `int' if <sys/types.h> does not define. */
-#undef ssize_t
-
-/* Define to `unsigned char' if <sys/types.h> does not define. */
-#undef uchar
-
-/* Define to `unsigned int' if <sys/types.h> does not define. */
-#undef uint
-
-/* Define to `unsigned long' if <sys/types.h> does not define. */
-#undef ulong
-
-/* Define to `unsigned long long' if <sys/types.h> does not define. */
-#undef ulonglong
-
-/* Define to `unsigned short' if <sys/types.h> does not define. */
-#undef ushort
-
-
 #else
 /* A configure-based build is not being used. */
 #include <jasper/jas_config2.h>
